src/ape/utils.py

```diff
@@ -154,7 +158,7 @@ def compute_checksum(source: bytes, algorithm: str = "md5") -> str:
     if algorithm == "md5":
         hasher = md5
     else:
-        raise Exception("Unknown algorithm")
+        raise ApeException(f"Unknown algorithm `{algorithm}`")
```

Comment:
Should avoid using the Base exception directly when raising (catching is fine as a catchall)

Reply:
Should be resolved now, everywhere. Thanks